root/static/scripts/edit/URLCleanup.js

```diff
@@ -1747,6 +1747,22 @@ const CLEANUPS = {
       return /^https:\/\/myspace\.com\//.test(url);
     },
   },
+  'napster': {
+    match: [new RegExp('^(https?://)?[a-z]{2}\\.napster\\.com', 'i')],
+    type: LINK_TYPES.streamingpaid,
+    clean: function (url) {
+      return url.replace(/^http:\/\//, 'https://');
+    },
```

Comment:
Does it make sense to keep the locale code subdomain?
That is, are these different stores or just localized interfaces?

Reply:
Dunno. Different sites have different language dropdowns (or none at all) - us. has a Spanish version and ar. has also Portuguese for some reason. So it's not just a way to translate stuff. But I don't know if they have different content. That said, https://ar.napster.com/artist/bread and https://us.napster.com/artist/bread have different numbers in the "and X other albums" link at the bottom, suggesting different content?

Reply:
I just found in the page footer that subdomains match country code, e.g. from https://es.napster.com/privacypolicy
>  Austria Dinamarca Finlandia Francia Alemania Grecia Irlanda Italia Luxemburgo Países Bajos Noruega Portugal España Suecia Suiza R. U. EE. UU.

So it seems alright to assume these are local stores with potential (even if small) differences.